feat(tem): remove 48h from faq (#5444)
scaleway/docs-content: pages/transactional-email/how-to/manage-blocklists.mdx

The temporary blocklist entry is created automatically to prevent repeated attempts to send emails, which could damage your reputation.

<Message type="important">

>   The duration of a temporary blocklist depends on whether or not the an automatic blocklist has already been created for the recipient email. If it is the first blocklist automatically created for the email, the blocklist duration is of 1 week. If a blocklist was already automatically created for the email once, blocklist duration is of 1 month and if three or more time, the duration is unlimited.

Reviewer: ```suggestion
  The duration of a temporary blocklist depends on whether or not an automatic blocklist has already been created for the recipient email in the past. If it is the first blocklist automatically created for the email, the blocklist duration is of 1 week. If a blocklist has been automatically created for the email once before, blocklist duration is of 1 month. If it is the third automatic blocklist or higher, the duration is unlimited.
```
